modules/aleph/includes/aleph.reservation.inc

```diff
@@ -72,21 +72,6 @@ function aleph_reservation_options($account, $default_value = '') {
  *   Form values.
  */
 function aleph_reservation_options_submit($account, array $values) {
-  // Load profile2 profile.
```

Comment:
Why implement the provider function at all? Don't the calling code check for its existence before invoking it?

Reply:
And - if we have to leave it in, consider adding a comment saying that it's left blank on purpose as we don't have anything to add.

Reply:
Sometimes. A lot of them are needed because the code doesn't check for it. 
I'll check if it works without it.